Removed invalid 'image-' prefixes

dstj opened the pull request:
registry-login-server, registry-username, registry-password do not have the 'image-' prefix

Versions:
azure-cli (2.0.12)
...
container (0.1.7)

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dstj committed:
Removed invalid 'image-' prefixes

registry-login-server, registry-username, registry-password do not have the 'image-' prefix

Versions:
azure-cli (2.0.12)
...
container (0.1.7)
